Makefile

```
@@ -52,7 +52,9 @@ endif
 govet:
 ifneq ($(HAS_SERVER),)
 	@echo Running govet
-	@$(GO) vet -shadow $$(go list ./server/...) || exit 1
+	$(GO) get -u golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow
+	$(GO) vet $(GOFLAGS) $$(go list ./server/...) || exit 1
+	$(GO) vet -vettool=$(which shadow) $$(go list ./server/...) || exit 1
```

@lieut-data provided a PR to change this on the server. (https://github.com/mattermost/mattermost-server/pull/10389) Should this also be changed here?

---

Yes! I found that `$(which shadow)` doesn't actually find `shadow` at all to my dismay (at least in the context of the `Makefile` -- bash works fine). So I've hard-coded the path, and improved other things in the linked PR.

I'm also debating the `-u` in the `go get` -- it makes the process 1s slower than it feels like it ought to be. Thoughts about not forcing the upgrade each time?

---

Let's continue this discussion on https://github.com/mattermost/mattermost-server/pull/10389/

---

@cpanato Please update to 

```suggestion
	$(GO) vet -vettool=$(GOPATH)/bin/shadow $$(go list ./server/...) || exit 1
```

as discussed in https://github.com/mattermost/mattermost-server/pull/10389